srml/grandpa/src/lib.rs

```diff
@@ -151,6 +152,13 @@ decl_storage! {
 
 		/// `true` if we are currently stalled.
 		Stalled get(stalled): Option<(T::BlockNumber, T::BlockNumber)>;
+
+		/// The number of changes (both in terms of keys and underlying economic responsibilities)
+		/// in the "set" of Grandpa validators from genesis.
+		CurrentSetId get(current_set_id) build(|_| fg_primitives::SetId::default()): SetId;
```

Comment:
```suggestion
		CurrentSetId get(current_set_id): SetId;
```

^^^ this is strictly equivalent.